editor/settings/editor_settings.cpp

```diff
@@ -1845,6 +1848,8 @@ float EditorSettings::get_auto_display_scale() {
 	}
 	return 1.0;
 #endif
```

Comment:
```suggestion
#endif // defined(WINDOWS_ENABLED)
```
Just for readability, as they are nested and far from where the if statement is